apps/entry/enums.py

```diff
@@ -46,5 +47,6 @@
     IDENTIFIER=IdentifierGrapheneEnum,
     FIGURE_CATEGORY_TYPES=FigureCategoryTypeEnum,
     FIGURE_TERMS=FigureTermsEnum,
-    SOURCES_RILIABILITY=FigureSourcesReliabilityEnum
+    SOURCES_RILIABILITY=FigureSourcesReliabilityEnum,
+    FigureReviewStatus=FigureReviewStatusEnum,
```

Comment:
```suggestion
    FIGURE_REVIEW_STATUS=FigureReviewStatusEnum,
```
Should this also use the similar convention? @thenav56

Reply:
I used FIGURE_REVIEW_STATUS